lib/Minz/Request.php

```diff
@@ -307,8 +359,41 @@ private static function magicQuotesOff() {
 		}
 	}
 
+	/**
+	 * @return array
+	 */
+	private static function extractHeaders() {
+		$headers = array();
+		foreach ($_SERVER as $key => $value) {
+			$headers[$key] = $value;
+		}
+		return $headers;
+	}
+
+	/**
+	 * @return mixed
+	 */
+	public static function getHeader($header, $default = null) {
+		if (array_key_exists($header, static::$headers)) {
```

Comment:
Any reason for using `array_key_exists` instead of the faster `isset`, which also has the advantage of dealing with null values (which I do not think we want to deal with manually in our case)?